src/superphot_plus/file_utils.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
+
+
+def read_single_lightcurve(filename, t0_lim=None):
+    """
+    Import a compressed lightcurve data file.
+
+    Parameters
+    ----------
+    filename : str
+        Name of the data file.
+    t0_lim : float, optional
```

Comment:
The name is a little confusing. The code includes everything before `t0_lim`, which doesn't match my mental model of t0 being a first time. I'm not sure if this is an artifact of what is stored in the light curve (there is a t0 list) or if we could call this `t_lim`.

Okay... reading down I think this makes sense. T0 will be the point of maximum flux and this is limiting when that can be. If that is the case, I think it is worth putting it in the comment.

Reply:
Renamed to ceiling, since that's more in-line with what's happening. All those points that are after the limit are removed from the return values, so it's not *only* used to limit the time of max flux.